src/Item/Item.service.ts

```diff
@@ -106,4 +110,113 @@ export class ItemService {
       await this.deleteDCLItem(dbItem)
     }
   }
+
+  private checkIfItemIsMovedToAnotherCollection(
```

Comment:
We can maybe simplify this method to
`checkItemMovedToAnotherCollection`
so we can remove `If` from the method name